web/packages/teleport/src/components/ReAuthenticate/ReAuthenticate.tsx

```diff
@@ -78,7 +78,8 @@ export function ReAuthenticate({
           <DialogHeader style={{ flexDirection: 'column' }}>
```

Comment:
I noticed that pressing Enter after typing the OTP doesn't submit the form. AFAIR, a quick fix for that would be to wrap the body of this component in `<form>` and set the type of the "Continue" button to `submit`.